lib/commands/download_images_from_args.dart

```diff
@@ -0,0 +1,163 @@
+
+import 'package:args/command_runner.dart';
+import 'package:figma_vars_to_dart/services/logger/logger.dart';
+
+import '../services/services.dart';
+
+class DownloadImagesFromArgsCommand extends Command {
+  final FigmaService figmaApi;
+  final ParserService parser;
+  final WriterService writer;
+  final CodeGeneratorService generator;
+  final LoggerService logger;
+
+  DownloadImagesFromArgsCommand({
+    required this.figmaApi,
+    required this.parser,
+    required this.writer,
+    required this.generator,
+    required this.logger,
+  }) {
+    argParser.addOption(
+      'fileId',
+      mandatory: true,
+      help: 'The id of the figma file, can be found in the URL.',
+    );
+    argParser.addOption(
+      'token',
+      mandatory: true,
+      help: 'Your Figma personal access token.',
+    );
+    argParser.addOption(
+      'outputFolder',
+      mandatory: false,
+      defaultsTo: 'assets',
+      help: 'Folder for the downloaded images, defaults to /assets.',
+    );
+    argParser.addOption(
+      'scale',
+      mandatory: false,
+      defaultsTo: '1.0,2.0,3.0',
+      help:
+          'Specify the scales of images to download (comma-separated), must be a number between 0.01 and 4. Defaults to 1.0,2.0,3.0. Warning: The "scale" argument is ignored for SVG format.',
+    );
+    argParser.addOption('format',
+        mandatory: false,
+        defaultsTo: 'png',
+        allowed: ['jpg', 'png', 'svg'],
+        help: 'Format of the downloaded images, defaults to png.');
+    argParser.addOption('section',
+        mandatory: false,
+        defaultsTo: null,
+        help:
+            r'Specify sections to download (comma-separated), defaults to all APP_ASSET_$section sections.');
+    argParser.addFlag('force',
+        defaultsTo: false,
+        help:
+            'Force download the images even if they already exist, defaults to false.');
+  }
+
+  @override
+  String get description =>
+      'Download images from Figma sections using command line args.';
+
+  @override
+  String get name => 'download-images';
+
+  @override
+  FutureOr? run() async {
+    final args = argResults;
+    if (args == null) {
+      return;
+    }
+
+    final fileId = args['fileId'] as String;
+    final token = args['token'] as String;
+    final outputFolder = args['outputFolder'] as String;
+    final imageFormat = args['format'] as String;
+    final scales = args['scale'] as String;
+    final forceDownload = args['force'] as bool;
+
+    List<String> sectionsToDownload = [];
+    if (args['sectionSuffix'] != null) {
+      sectionsToDownload = (args['sectionSuffix'] as String)
+          .split(',')
+          .map((e) => e.trim())
+          .toList();
+    }
+
+    List<double> scalesList = handleScales(imageFormat, scales);
+
+    logger.log('\n📥 Fetching Figma file...');
+    var fileData = await figmaApi.fetchFigmaFile(fileId: fileId, token: token);
+
+    print("📂 Identifying 'APP_ASSET_' sections...");
```

Comment:
look like this way we would be able to decouple logs better, same for other `print`
```suggestion
    logger.log("📂 Identifying 'APP_ASSET_' sections...");
```